### PR TITLE
Feature calculator

### PR DESCRIPTION
**Pull request by serengia**
Hello. Kindly approve the request to merge feature-calculator to development.
The branch contains the implementation of the following:
- Basic initial implementation of calculator component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+      </div>
+
+      <footer className="footer">
+        <div className="footer-container row">
+          <h4>Footer</h4>
+        </div>
+      </footer>
     </div>
   );
 }
```
Comment (ichala) [inline src/App.js:26]: - [ ] as part of the requirements the App.js & Calculator.js components should be a class-based component , kindly fix that 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️
- Good Readme file ✔️  


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (serengia) [reply src/App.js:26]: Hello @ichala . Sorry, I did not see that in the requirements. Instead, they insisted we use functional components. Kindly advice. https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md

**Review by Munsa1 — APPROVED** [review]
Hi @serengia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**